polyfill.js

```diff
@@ -0,0 +1,137 @@
+'use strict'
+
+if (typeof Symbol !== 'function' || !Symbol.isConcatSpreadable) {
+  throw new Error('Symbol.isConcatSpreadable missing from the environment')
+}
+
+if (typeof Array.prototype.flatMap !== 'function') {
+  // eslint-disable-next-line no-extend-native
+  Object.defineProperty(Array.prototype, 'flatMap', {
+    enumerable: false,
+    // https://tc39.github.io/proposal-flatMap/#sec-Array.prototype.flatMap
+    value: function flatMap (callback, thisArg = undefined) {
```

Comment:
Do we need a build step, or to otherwise convert these (defaults, templates) to ES5?